src/server/lib/validation-error.js

```diff
@@ -0,0 +1,22 @@
+
+class ValidationError extends Error {
```

Comment:
Just curious: Is this the file you had to extract to break the circular dependency?

Reply:
yup